docs/source/reference/text_classification.rst

```diff
@@ -126,19 +68,31 @@ By default, we use the `bert-base-uncased <https://arxiv.org/abs/1810.04805>`_ m
 
 .. note:: When changing the backbone, make sure you pass in the same backbone to the Task and the Data object!
 
-.. code-block:: python
+.. testsetup::
 
-    datamodule = TextClassificationData.from_files(
+    from flash.data.utils import download_data
+    from flash.text import TextClassificationData, TextClassifier
+
+    download_data("https://pl-flash-data.s3.amazonaws.com/imdb.zip", "data/")
+
+.. testcode::
+
+    datamodule = TextClassificationData.from_csv(
+        "review",
+        "sentiment",
     	backbone="bert-base-chinese",
         train_file="data/imdb/train.csv",
-        valid_file="data/imdb/valid.csv",
-        input="review",
-        target="sentiment",
+        val_file="data/imdb/valid.csv",
         batch_size=512
     )
 
     task = TextClassifier(backbone="bert-base-chinese", num_classes=datamodule.num_classes)
 
+.. testoutput::
```

Comment:
What does it do ?

Reply:
just accept any output from the previous code block

Reply:
This just says to ignore the output. Otherwise the test is assumed to expect no outputs, but all the huggingface logging and stuff causes it to fail